assets/javascripts/redmine_s3.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * This script handles thumbnail image missing errors
+ */
+
+$(function() {
```

Comment:
So how about this? I will add generate thumbnails on image upload feature as you suggest, but I would like to have script too because otherwise if the thumbnail is missing for some reason (for example, we cleaned the tmp folder) it will never be generated again.